pocs/tests/conftest.py

```diff
@@ -22,19 +23,38 @@ def images_dir(tmpdir_factory):
 
 
 @pytest.fixture(scope='function')
-def config(images_dir):
+def config(images_dir, messaging_ports):
     pocs.base.reset_global_config()
 
     global _one_time_config
     if not _one_time_config:
         _one_time_config = load_config(ignore_local=True, simulator=['all'])
+        # Set several fields to fixed values.
         _one_time_config['db']['name'] = 'panoptes_testing'
         _one_time_config['name'] = 'PAN000'  # Make sure always testing with PAN000
         _one_time_config['scheduler']['fields_file'] = 'simulator.yaml'
 
-    _one_time_config['directories']['images'] = images_dir
+    # Make a copy before we modify based on test fixtures.
+    result = copy.deepcopy(_one_time_config)
 
-    return copy.deepcopy(_one_time_config)
+    # We allow for each test to have its own images directory, and thus
+    # to not conflict with each other.
+    result['directories']['images'] = images_dir
```

Comment:
Ah, good, thanks. (even though not part of PR)

Reply:
OK, fine, I did do a little straightening up and commenting while I was at it! ;-)